packages/harpguru-core/src/components/harp-cell-inaccessible/harp-cell-inaccessible.tsx

```diff
@@ -0,0 +1,14 @@
+import { View, ViewStyle } from 'react-native'
+import React from 'react'
+
+type StyleProps = {
+  readonly baseStyles: ViewStyle
+}
+
+export const HarpCellInaccessible = ({
+  baseStyles,
+}: StyleProps): React.ReactElement => {
+  return <View accessible={false} style={baseStyles} />
+}
+
+export const MemoHarpCellInaccessible = React.memo(HarpCellInaccessible)
```

Comment:
Need to double check whether the memoisation is actually working here due to the deep style object included in the component params.

Reply:
Done. It wasn't. Now it is again.